### PR TITLE
Validate validation-overrides.xml when validating model

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: config-model-api/src/main/java/com/yahoo/config/application/api/ValidationOverrides.java [hunk 1]
```diff
@@ -73,15 +73,24 @@ public boolean allows(String validationIdString, Instant now) {
 
     /** Returns whether the given (assumed invalid) change is allowed by this at the moment */
     public boolean allows(ValidationId validationId, Instant now) {
+        validate(now);
```
Comment (bratseth) [inline config-model-api/src/main/java/com/yahoo/config/application/api/ValidationOverrides.java:76]: Shouldn't this line be removed?

**Review by bratseth — CHANGES_REQUESTED** [review]

File: config-model-api/src/main/java/com/yahoo/config/application/api/ValidationOverrides.java (hunk 1 above)
Comment (hmusum) [reply config-model-api/src/main/java/com/yahoo/config/application/api/ValidationOverrides.java:76]: Oops, yes, that should not be there

Comment (hmusum) [reply config-model-api/src/main/java/com/yahoo/config/application/api/ValidationOverrides.java]: Fixed

**Review by bratseth — APPROVED** [review]
👍

**Pull request merged**